src/transformers/pipelines.py

```diff
@@ -1391,6 +1391,10 @@ class SummarizationPipeline(Pipeline):
             on the associated CUDA device id.
     """
 
+    def __init__(self, task: str = "summarization", **kwargs):
```

Comment:
A string is mutable and it's generally a [bad idea](https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/#:~:text=In%20Python%2C%20when%20passing%20a,or%20even%20a%20class%20instance.) to have default values be mutable in Python.

I'd do `task=None` here and a test to set it to "summarization" though there are quite a few of those string default args in this module so maybe it's okay. It doesn't look like task is ever mutated in any case, so it's probably safe.

Reply:
Does it even need to be an argument?
Is `SummarizationPipeline` meant to support other tasks?